pandapower/converter/cim/cim2pp/converter_classes/transformers/powerTransformersCim16.py

```diff
@@ -473,25 +502,12 @@ def _prepare_trafos_cim16(self, power_trafo2w: pd.DataFrame) -> pd.DataFrame:
         power_trafo2w['vk0_percent'] = power_trafo2w['x0_sign'] * power_trafo2w['vk0_percent']
         power_trafo2w['std_type'] = None
         power_trafo2w['df'] = 1.
-        # todo remove if pp supports phase shifter
-        if power_trafo2w.loc[power_trafo2w['angle'].notna()].index.size > 0:
-            self.logger.warning("Modifying angle from 2W transformers. This kind of angle regulation is currently not "
-                                "supported by pandapower! Affected transformers: \n%s" %
-                                power_trafo2w.loc[power_trafo2w['angle'].notna()])
-            self.cimConverter.report_container.add_log(Report(
-                level=LogLevel.WARNING, code=ReportCode.WARNING_CONVERTING,
-                message="Modifying angle from 2W transformers. This kind of angle regulation is currently not "
-                        "supported by pandapower! Affected transformers: \n%s" %
-                        power_trafo2w.loc[power_trafo2w['angle'].notna()]))
-        power_trafo2w['phaseAngleClock_temp'] = power_trafo2w['phaseAngleClock'].copy()
-        power_trafo2w['phaseAngleClock'] = power_trafo2w['angle'] / 30
-        power_trafo2w['phaseAngleClock'] = power_trafo2w['phaseAngleClock'].fillna(power_trafo2w['angle_lv'] * -1 / 30)
-        power_trafo2w['phaseAngleClock'] = power_trafo2w['phaseAngleClock'].fillna(power_trafo2w['phaseAngleClock_temp'])
+        power_trafo2w.loc[power_trafo2w['phaseAngleClock'] == 0, 'phaseAngleClock'] = np.nan
         power_trafo2w['phaseAngleClock_lv'] = power_trafo2w['phaseAngleClock_lv'].fillna(0)
         power_trafo2w['shift_degree'] = power_trafo2w['phaseAngleClock'].astype(float).fillna(
             power_trafo2w['phaseAngleClock_lv'].astype(float)) * 30
         power_trafo2w['parallel'] = 1
-        power_trafo2w['tap_phase_shifter'] = False
+        # power_trafo2w['tap_phase_shifter'] = False  # removed
```

Comment:
delete commented out code? github has the history

Reply:
deleted